packages/pages-react/studies/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Why create another package pages. they other one is not called core-components its core-react I would put this one inside code-react or renamed it core-components